http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -653,7 +653,14 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
                   )
                 )
             }
-          case None => deleteDiskAction
+          case None =>
+            // Error'd runtimes might not have a WSM resourceId and therefore no WsmJobStatus.
```

Comment:
Trying to fix this scenario which I think is currently happening to a few users:

1. User creates runtime
2. Back Leo processes `createRuntime` request
3. WSM call fails; Leo moves runtime to `Error` status
4. User tries to delete Error'd runtime
5. Front Leo moves runtime to `Deleting` status
6. Back Leo treats it as a no-op and doesn't fail; but never actually moves it to `Deleted` status